README.md

```diff
@@ -1 +1,55 @@
-# Book_list_ES6
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Book list
+
+
+> This web app is about making a book list
+
+> This site contans
+>- An add form that adds the books
+>- A remove button that is attached to each list item removs the book from the list
+>- There is a date and time at the buttom of the page
+
+![screenshot](./img/Screenshot.png)
+
+
+Additional description about the project and its features.
+
+## live demo
+
+- project: (https://eng-mohamed-eldeeb.github.io/Book_list_ES6/)
```

Comment:
- [ ] Please, consider that your demo live link is not working correctly:
<img width="695" alt="Screen Shot 2022-02-22 at 00 35 56" src="https://user-images.githubusercontent.com/49064106/155023224-0595ed97-ef22-43fb-83c8-e69ac7cdab28.png">